torch/csrc/jit/passes/quantization.h

```diff
@@ -122,5 +122,49 @@ TORCH_API script::Module InsertObservers(
     const script::Module& observer_module,
     const script::Module& weight_observer_module);
 
+/** \brief Insert quantize - int_repr - dequantize calls to the Tensors
+ *  that are observed in insert_observers pass
+ *
+ * For each Tensor that is observed, get the observer module and call
+ * calculate_qparam on the observer module to get quantization parameters
+ * and add quantize - int_repr - dequantize function calls using these parameters
+ * we also have special handling for quantizing "bias" right now.
+ *
+ * \param module the input module
+ * \param method_name the method we want to insert quantization calls for
+ */
+TORCH_API void InsertQuantDeQuant(
+    script::Module& module,
+    const std::string& method_name);
+
+/** \brief Backend specific pass to fuse dequantize - op - quantize calls
+ * as quantized_op calls.
+ *
+ * Right now this is a fusion for fbgemm backend and only works for quantized
+ * conv op, we'll extend to more ops and more backends in the future.
+ *
+ * \param graph the graph we want to apply fusion
+ */
+TORCH_API void QuantFusion(std::shared_ptr<Graph>& graph);
+
+class QuantizeHelper {
```

Comment:
Nit: since this class is not for the public use, we should move it to the cpp file.